### DISCUSSION TITLE
rubygems.org api for the dump of metadata of all gems

### DISCUSSION THREAD

**Discussion by nareshdevasani** [ruby/rubygems#6398] [Q&A] [answered]
Dear all,

I am looking for an API which can download the complete dump of all the ruby gems in one call.

If not available, how can I get the complete list of gems (names)?

Thanks,
Naresh

**Comment by simi** [comment, marked as answer]
You can grab "anonymized" DB dump at https://rubygems.org/pages/data. It includes all gem names.
